code/modules/paperwork/desk_bell.dm

```diff
@@ -0,0 +1,96 @@
+// A receptionist's bell
+
+/obj/item/desk_bell
+	name = "desk bell"
+	desc = "The cornerstone of any customer service job. You feel an unending urge to ring it."
+	icon = 'icons/obj/objects.dmi'
+	icon_state = "desk_bell"
+	w_class = SIZE_SMALL
+	embeddable = FALSE
+	/// The amount of times this bell has been rang, used to check the chance it breaks.
+	var/times_rang = 0
+	/// Is this bell broken?
+	var/broken_ringer = FALSE
+	/// Holds the time that the bell can next be rang.
+	COOLDOWN_DECLARE(ring_cooldown)
+	/// The length of the cooldown. Setting it to 0 will skip all cooldowns alltogether.
+	var/ring_cooldown_length = 5 SECONDS // This is here to protect against tinnitus.
+	/// The sound the bell makes.
+	var/ring_sound = 'sound/misc/desk_bell.ogg'
+
+/obj/item/desk_bell/attack_hand(mob/living/user)
+	if(!ishuman(user))
+		return FALSE
+	if(!anchored)
+		return ..()
+	if(!COOLDOWN_FINISHED(src, ring_cooldown))
+		return FALSE
+	if(!ring_bell(user))
+		to_chat(user, SPAN_NOTICE("[src] is silent. Some idiot broke it."))
+		return FALSE
+	return TRUE
+
+/obj/item/desk_bell/MouseDrop(atom/over_object)
+	var/mob/mob = usr
+	if(!Adjacent(mob) || anchored)
+		return
+	if(!ishuman(mob))
+		return
+
+	if(over_object == mob)
+		mob.put_in_hands(src)
+
+/obj/item/desk_bell/attackby(obj/item/item, mob/user)
+	//Repair the desk bell if its broken and we're using a screwdriver.
+	if(HAS_TRAIT(item, TRAIT_TOOL_SCREWDRIVER))
+		if(broken_ringer)
+			visible_message(SPAN_NOTICE("[user] begins repairing [src]..."), SPAN_NOTICE("You begin repairing [src]..."))
```

Comment:
It looks like the `visible_message()` needs to be from the user here, otherwise this message isn't visible when you repair it in your hand. (This isn't a problem for the others thanks to the `isturf()` check below.)
```suggestion
			user.visible_message(SPAN_NOTICE("[user] begins repairing [src]..."), SPAN_NOTICE("You begin repairing [src]..."))
```